src/modules/order/components/DiscountContainer/DiscountContainer.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from '@emotion/react';
```

Comment:
I suggest export styles as a default export instead of exporting every style variable by itself.
That way you will have no clashes when importing styles in different components and will be able to clearly separate style variables from other variables.

Remove exports from constants and export the styles as `export default { discountContainer, discountCode, ...}`. You can import the styles in the component with `import styles from "./DiscountContainer.styles` and can use it in the view with `<div css={styles.discountContainer}>...</div>` etc.